src/spireslayer/save_editor.py

```diff
@@ -8,7 +8,7 @@
 
 class SaveEditor(object):
     def __init__(self,
-                 save_file_path: str = "C:\\Program Files (x86)\\Steam\\steamapps\\common\\SlayTheSpire\\saves",
+                 save_file_path: str = "/path/to/save/folder",
```

Comment:
I agree, the change to the save folder path was mainly because i was playing on a OS device (confirmed compatibility, i guess).

The default path its well. People who install in a different partition/hard drive will need to change that line, but its manageable :)

Thanks for the review.